src/CoreBundle/private/library/classes/TCMSListManager/TCMSListManagerMediaSelector.class.php

```diff
@@ -205,12 +205,6 @@ protected function GetMaxHeight()
 
         if ($this->oGlobal->UserDataExists('imageHeight')) {
             $iHeight = $this->oGlobal->GetUserData('imageHeight');
-        } else {
-            if (isset($this->oDefaultImage->aData) && is_array($this->oDefaultImage->aData)) {
-                if (array_key_exists('height', $this->oDefaultImage->aData) && $this->oDefaultImage->aData['height'] > 0) {
-                    $iWidth = $this->oDefaultImage->aData['height'];
```

Comment:
Why is this removed?

The wrong variable name ($iWidth instead of $iHeight) shows that this was not used?

Reply:
Ah right, I didn't see that the wrong name led to the unused-message. This has been unchanged for almost 9 years now, so I'm inclined to just keep it removed, as no one seems to complain.

Reply:
The documentation should be adapted then.

Reply:
Done.